### PR TITLE
Creating Strings python solution

### PR DESCRIPTION
**Pull request by maggie-j-liu**
I added a python solution using `itertools.permutations` to Creating Strings. 

Maybe we should change the `next_permutation` section to a LanguageSection, and put this section there? I wasn't sure so I put it in a separate section for now, but I can change it.

### CODE REVIEW

**Comment by dongliu0426** [comment]
> Maybe we should change the next_permutation section to a LanguageSection, and put this section there?

👍

**Review by dongliu0426 — APPROVED** [review]

File: content/2_Bronze/Complete_Rec.mdx [hunk 1]
```diff
@@ -779,15 +750,28 @@ int main() {
...
+Since `itertools.permutations` treats elements as unique based on position, not
+value, it returns all permutations, with repeats. Putting the returned tuples in
+a set can filter out duplicates, and since tuples are returned, we need to join
+the characters into a string.
 
-</Warning>
+```py
+import itertools
+s = input()
+# perms is a sorted list of all the permutations
```
Comment (SansPapyrus683) [inline content/2_Bronze/Complete_Rec.mdx:765]:
```suggestion
# perms is a sorted list of all the permutations of the given string
```

File: content/2_Bronze/Complete_Rec.mdx [hunk 2]
```diff
@@ -122,7 +122,8 @@ public class Main {
 			return Math.abs(s1_sum - s2_sum);
 		}
 
-		return Math.min(solve(index+1, s1_sum + weights[index], s2_sum), solve(index+1, s1_sum, s2_sum+weights[index]));
+		return Math.min(solve(index + 1, s1_sum + weights[index], s2_sum),
+						solve(index + 1, s1_sum, s2_sum + weights[index]));
```
Comment (SansPapyrus683) [inline content/2_Bronze/Complete_Rec.mdx:126]: format this like
```java
Math.min(
    solve(index + 1, s1_sum + weights[index], s2_sum),
    solve(index + 1, s1_sum, s2_sum + weights[index])
);
```

File: content/2_Bronze/Complete_Rec.mdx [hunk 3]
```diff
@@ -1015,10 +1029,10 @@ void dfs(int r) { // place queen in r-th row
 		++ans; // found valid placement
 		return;
 	}
-	for (int i = 0; i < 8; ++i) if (g[r][i] == '.' && !c[i] && !sum[i+r] && !dif[i-r+7]) {
-		c[i] = sum[i+r] = dif[i-r+7] = 1;
-		dfs(r+1);
-		c[i] = sum[i+r] = dif[i-r+7] = 0;
+	for (int i = 0; i < 8; ++i) if (g[r][i] == '.' && !c[i] && !sum[i + r] && !dif[i - r + 7]) {
```
Comment (SansPapyrus683) [inline content/2_Bronze/Complete_Rec.mdx:1032]: can you put the if statement in its own little block with curly braces and everything

File: content/2_Bronze/Complete_Rec.mdx [hunk 4]
```diff
@@ -572,7 +573,7 @@ void search() {
 int main() {
 	setIO();
 	cin >> S;
-	for (auto c : S) cnt[c-'a'] ++;
+	for (auto c : S) cnt[c - 'a']++;
```
Comment (SansPapyrus683) [inline content/2_Bronze/Complete_Rec.mdx:576]: replace `auto` w/ `char`?

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]

File: content/2_Bronze/Complete_Rec.mdx [hunk 5]
```diff
@@ -122,8 +122,10 @@ public class Main {
 			return Math.abs(s1_sum - s2_sum);
 		}
 
-		return Math.min(solve(index + 1, s1_sum + weights[index], s2_sum),
-						solve(index + 1, s1_sum, s2_sum + weights[index]));
+		return Math.min(
```
Comment (SansPapyrus683) [inline content/2_Bronze/Complete_Rec.mdx:125]: the formatting looks a bit weird- did you use spaces or something?

**Review by SansPapyrus683 — APPROVED** [review]
looks good enough to pass

File: content/2_Bronze/Complete_Rec.mdx (hunk 5 above)
Comment (maggie-j-liu) [reply content/2_Bronze/Complete_Rec.mdx:125]: looks like this: 
image
is that okay?

**Review by HarryWangATX — APPROVED** [review]

**Pull request merged**